texttospeech/quickstart/quickstart.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
```

Comment:
Sorry, please add a comment for this one, too.

Reply:
Added